XLS-0073-disllow-frozen-trustline-receiving/README.md

```diff
@@ -0,0 +1,92 @@
+<pre>
+Title:       <b>Disallow Receiving on Frozen Trustlines</b>
+Type:        <b>draft</b>
+
+Author:      <a href="mailto:shawnxie@ripple.com">Shawn Xie</a>
+Affiliation: <a href="https://ripple.com">Ripple</a>
+</pre>
+
+#  Disallow Receiving on Frozen Trustlines
+
+## Abstract
+
+This amendment empowers token issuers on the XRP Ledger to regulate tokens in an effort to prevent misuse by blacklisted accounts. This document proposes enhancements to the interactions between frozen assets and payment. This proposal will ensure that blacklisted token holders can no longer receive funds, which will significantly improve use cases such as stablecoins, where ensuring regulatory compliance is essential.
```

Comment:
This paragraph uses "this proposal" and variants a lot, which IMO makes it a bit hard to read